plugins/source/shopify/client/transformers.go

```diff
@@ -1,27 +1,10 @@
 package client
 
 import (
-	"reflect"
-
-	"github.com/apache/arrow/go/v13/arrow"
 	"github.com/cloudquery/plugin-sdk/v3/schema"
 	"github.com/cloudquery/plugin-sdk/v3/transformers"
-	"github.com/cloudquery/plugin-sdk/v3/types"
 )
 
-func typeTransformer(field reflect.StructField) (arrow.DataType, error) {
-	switch reflect.New(field.Type).Elem().Interface().(type) {
-	case []any:
-		return types.ExtensionTypes.JSON, nil
-	default:
-		return nil, nil
-	}
-}
-
-var options = []transformers.StructTransformerOption{
-	transformers.WithTypeTransformer(typeTransformer),
-}
-
 func TransformWithStruct(t any, opts ...transformers.StructTransformerOption) schema.Transform {
-	return transformers.TransformWithStruct(t, append(options, opts...)...)
+	return transformers.TransformWithStruct(t, opts...)
```

Comment:
if we don't have any common options, just drop `TransformWithStruct` func & replace with direct call to `transformers.TransformWithStruct`